pkgs/tools/security/pass/extensions/checkup.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub
+, coreutils, curl, findutils, gnugrep, gnused }:
+
+stdenv.mkDerivation rec {
+  pname = "pass-checkup";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "etu";
+    repo = "pass-checkup";
+    rev = version;
+    sha256 = "17fyf8zj535fg43yddjww1jhxfb3nbdkn622wjxaai2nf46jzh7y";
+  };
+
+  buildInputs = [ coreutils curl findutils gnugrep gnused ];
```

Comment:
I don't think this is what you want. 

```
[nix-shell:~/.cache/nix-review/pr-71919]$ nix-store -q --tree ./results/passExtensions.pass-checkup
/nix/store/jikx5y2wb45r1cr0ar0j4ma4f76gh0mq-pass-checkup-0.2.0
+---/nix/store/zavn4np1jvm79f0rafkv0p1mrag09qkz-bash-4.4-p23
    +---/nix/store/qn76sklvyalzw9ilnxz6sh0020gl2qn6-glibc-2.27
    |   +---/nix/store/qn76sklvyalzw9ilnxz6sh0020gl2qn6-glibc-2.27 [...]
    +---/nix/store/zavn4np1jvm79f0rafkv0p1mrag09qkz-bash-4.4-p23 [...]
```
curl, grep and other won't be there in your closure by default. 

I'm not sure how pass works, but if this was a standalone script i would do:
```
  postInstall = ''
    wrapProgram $out/lib/password-store/extensions/checkup.bash \
      --set PATH ${ lib.makeBinPath [ coreutils curl findutils gnugrep gnused ] }
  '';
```

Reply:
unless the pass.withExtensions collects the buildInputs and exposes them in an environment

Reply:
still doesn't seem like that happens:

```
[nix-shell:~/.cache/nix-review/pr-71919]$ nix-shell --pure -p "with import ./nixpkgs {}; pass.withExtensions(ext:[ passExtensions.pass-checkup])"

[nix-shell:~/.cache/nix-review/pr-71919]$ curl --help
The program ‘curl’ is currently not installed. It is provided by
several packages. You can install it by typing one of the following:
  nix-env -iA nixos.curl
  nix-env -iA nixos.curlFull
```